hw2/eesss34690/q2/pytest.py

```diff
@@ -0,0 +1,19 @@
+import _vector as vec
```

Comment:
You unit-test is implemented wrong.

- You should rename your pytest.py yo test**.py. (file name must starts with test)
- Your unit test function should also start with test, otherwise, the test won't be triggered.

Reply:
You can take a look at this tutorial
https://semaphoreci.com/community/tutorials/testing-python-applications-with-pytest